docs/proxy.md

```diff
@@ -18,9 +18,9 @@ Follow the instructions below to let Cert Manager Operator trust a custom Certif
 
 2.  Consume the created configmap in Cert Manager Operator's deployment by updating its subscription:
 
-        ```bash
-        oc -n cert-manager-operator patch subscription cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}]}}}'
-        ```
+    ```bash
+    oc -n cert-manager-operator patch subscription openshift-cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}]}}}'
```

Comment:
@xingxingxia after giving this one a re-thought, IMO can we please keep this one as is in case of this GitHub repo and just update this to use "openshift-cert-manager-operator" only in https://github.com/openshift/openshift-docs 
The rationale behind it being: users installing the operator from GitHub source will continue to use subscription name as "cert-manager-operator" while those who installed the productized RH version of the operator from OperatorHub (rh operator-index) would have subscription object named "openshift-cert-manager-operator" (which is becuase we patch the operator package id in midstream before building the productized version).

Reply:
Sure, but I updated with placeholder "<subscription_name>"

Reply:
Yeah, that's a good idea :)